packages/framer-motion/src/motion/utils/use-visual-element.ts

```diff
@@ -21,8 +22,9 @@ export function useVisualElement<Instance, RenderState>(
     const presenceContext = useContext(PresenceContext)
     const reducedMotionConfig = useContext(MotionConfigContext).reducedMotion
 
-    const visualElementRef: MutableRefObject<VisualElement | undefined> =
-        useRef(undefined)
+    const visualElementRef: MutableRefObject<
+        VisualElement<Instance> | undefined
+    > = useRef(undefined)
```

Comment:
I think you can infer the type by doing 
```suggestion
    const visualElementRef = useRef<VisualElement<Instance>>(undefined)
```

Reply:
Thanks, changed this